### ISSUE TITLE
boostrap readme missing cloudresourcemanager.googleapis.com API enablement

### ISSUE THREAD

**Issue by obriensystems** [terraform-google-modules/terraform-example-foundation#965]
### TL;DR

terraform plan has errors on IAM permissions - 2 were my missing cloudidentity API enablement and roles/serviceusage.serviceUsageConsumer grant - but the 3rd was a missing cloudresourcemanager.googleapis.com API enablement

```
root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ terraform plan -input=false -out bootstrap.tfplan

  + gcs_bucket_tfstate                                = (known after apply)
  + networks_step_terraform_service_account_email     = (known after apply)
  + organization_step_terraform_service_account_email = (known after apply)
  + projects_gcs_bucket_tfstate                       = (known after apply)
  + projects_step_terraform_service_account_email     = (known after apply)
  + seed_project_id                                   = (known after apply)
╷
│ Error: Error when reading or editing Organization Not Found : 9....309: googleapi: Error 403: Cloud Resource Manager API has not been used in project 46....3 before or it is disabled. Enable it by visiting https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=46....63 then retry. If you enabled this API recently, wait a few minutes for the action to propagate to our systems and retry.
│ Details:
│ [
│   {
│     "@type": "type.googleapis.com/google.rpc.Help",
│     "links": [
│       {
│         "description": "Google developers console API activation",
│         "url": "https://console.developers.google.com/apis/api/cloudresourcemanager.googleapis.com/overview?project=46...3"
│       }
│     ]
│   },
│   {
│     "@type": "type.googleapis.com/google.rpc.ErrorInfo",
│     "domain": "googleapis.com",
│     "metadata": {
│       "consumer": "projects/46...63",
│       "service": "cloudresourcemanager.googleapis.com"
│     },
│     "reason": "SERVICE_DISABLED"
│   }
│ ]
│ , accessNotConfigured
│ 
│   with data.google_organization.org[0],
│   on groups.tf line 30, in data "google_organization" "org":
│   30: data "google_organization" "org" {
│ 
```

missed
Enable the Cloud Identity API (cloudidentity.googleapis.com) on the billing project.
Grant role roles/serviceusage.serviceUsageConsumer to the user running Terraform on the billing project.

on https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/README.md#optional---automatic-creation-of-google-cloud-identity-groups

```
root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ gcloud organizations add-iam-policy-binding "${ORGANIZATION_ID}" --member "user:${EMAIL}" --role roles/serviceusage.serviceUsageConsumer

root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ gcloud services enable cloudidentity.googleapis.com
Operation "operations/acat.p2-464286101163-44fe637f-13fb-4e86-abb9-8aa6fd3e5405" finished successfully.

```

verifying APIs enabled
```
root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ gcloud services list | grep NAME
NAME: bigquery.googleapis.com
NAME: bigquerymigration.googleapis.com
NAME: bigquerystorage.googleapis.com
NAME: cloudapis.googleapis.com
NAME: clouddebugger.googleapis.com
NAME: cloudidentity.googleapis.com
NAME: cloudtrace.googleapis.com
NAME: datastore.googleapis.com
NAME: logging.googleapis.com
NAME: monitoring.googleapis.com
NAME: servicemanagement.googleapis.com
NAME: serviceusage.googleapis.com
NAME: sql-component.googleapis.com
NAME: storage-api.googleapis.com
NAME: storage-component.googleapis.com
NAME: storage.googleapis.com
```
missing cloudresourcemanager

enabling
```
root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ gcloud services enable cloudresourcemanager.googleapis.com
Operation "operations/acat.p2-464286101163-ab55063b-66d9-4dfb-a6b2-016e8d8ac84a" finished successfully.
```

rerun terraform plan
```
root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ terraform plan -input=false -out bootstrap.tfplan

root_@cloudshell:~/lz-tef-plz/CloudLandingZone/terraform-example-foundation/0-bootstrap (lz-tef-plz)$ terraform plan -input=false -out bootstrap.tfplan

        }
      + monitoring_workspace_users = {
          + id            = "monitoring_workspace_users@pbmm.landing.systems"
          + name          = "monitoring_workspace_users"
          + resource_name = (known after apply)
        }
    }
  + seed_project_id                                   = (known after apply)

─────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────

Saved the plan to: bootstrap.tfplan

To perform exactly these actions, run the following command to apply:
    terraform apply "bootstrap.tfplan"

### Expected behavior

terraform plan - finishes

### Observed behavior

terraform plan stops on missing API enablement above

### Terraform Configuration

```hcl
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ terraform version
Terraform v1.4.6
on linux_amd64
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ gcloud version
Google Cloud SDK 428.0.0
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ git version
git version 2.30.2

all good (above 1.3.0, 393, 2.28)
```

### Terraform Version

```sh
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ terraform version
Terraform v1.4.6
on linux_amd64
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ gcloud version
Google Cloud SDK 428.0.0
root_@cloudshell:~/lz-tef-plz/CloudLandingZone (lz-tef-plz)$ git version
git version 2.30.2

all good (above 1.3.0, 393, 2.28)
```

### Additional information

I'll put a patch in shortly

**Comment by obriensystems** [comment]
part of #940

**Comment by fmichaelobrien** [comment]
Assigned to fmichaelobrien

**Comment by fmichaelobrien** [comment]
Fmichaelobrien
